### PR TITLE
Bump npm packages

### PR DESCRIPTION
**Pull request by vvalleru**
<!-- Available types:
- fix: Bump npm version and packages to fix dependabot issues.

Issue #, if available:
* https://github.com/aws/amazon-genomics-cli/security/dependabot/11
* https://github.com/aws/amazon-genomics-cli/security/dependabot/3
* https://github.com/aws/amazon-genomics-cli/security/dependabot/10
* https://github.com/aws/amazon-genomics-cli/security/dependabot/4
* https://github.com/aws/amazon-genomics-cli/security/dependabot/2
* https://github.com/aws/amazon-genomics-cli/security/dependabot/13

**Description of Changes**

[//]: #  (A description of the change that you made and the new user experience that it creates)

**Description of how you validated changes**

[//]: #  Run `npm audit` to make sure no vulnerabilities are present.

```
(22-11-07 16:56:27) <0> [~/Workplace/amazon-genomics-cli/scripts]
3c22fb0f52bd % npm audit

                       === npm audit security report ===

found 0 vulnerabilities
 in 8 scanned packages

(22-11-07 16:56:31) <0> [~/Workplace/amazon-genomics-cli/scripts]
3c22fb0f52bd % cd ../extras/agc-minimal-permissions

(22-11-07 16:56:40) <0> [~/Workplace/amazon-genomics-cli/extras/agc-minimal-permissions]
3c22fb0f52bd % npm audit

                       === npm audit security report ===

found 0 vulnerabilities
 in 428 scanned packages

(22-11-07 16:56:43) <0> [~/Workplace/amazon-genomics-cli/extras/agc-minimal-permissions]
3c22fb0f52bd %
```

**Checklist**

- [N/A] If this change would make any existing documentation invalid, I have included those updates within this PR
- [N/A] I have added unit tests that prove my fix is effective or that my feature works
- [N/A] I have linted my code before raising the PR
- [Y] Title of this Pull Request follows Conventional Commits standard: https://www.conventionalcommits.org/en/v1.0.0/

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: extras/agc-minimal-permissions/package.json [hunk 1]
```diff
@@ -11,19 +11,23 @@
     "cdk": "cdk"
   },
   "devDependencies": {
-    "@aws-cdk/assert": "1.126.0",
+    "@aws-cdk/assert": "^2.50.0",
     "@types/jest": "^27.0.2",
     "@types/node": "16.10.3",
-    "aws-cdk": "^1.126.0",
+    "aws-cdk": "2.50.0",
```
Comment (lingjiekong) [inline extras/agc-minimal-permissions/package.json:17]: nit: should we use `^2.50.0` to get latest version?

File: scripts/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "aws-cdk": "^1.130.0",
+    "aws-cdk": "2.50.0",
```
Comment (lingjiekong) [inline scripts/package.json:3]: same as above.

**Review by lingjiekong — APPROVED** [review]
approve with minor comment

File: extras/agc-minimal-permissions/package.json (hunk 1 above)
Comment (vvalleru) [reply extras/agc-minimal-permissions/package.json]: Updated

File: scripts/package.json (hunk 2 above)
Comment (vvalleru) [reply scripts/package.json]: Updated

**Review by lingjiekong — APPROVED** [review]
LGTM!

**Pull request merged**